src/components/generic/error-handler.tsx

```diff
@@ -0,0 +1,53 @@
+import {
+  AppBar,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  IconButton,
+  Toolbar,
+  Typography,
+} from "@mui/material";
+import { GridCloseIcon } from "@mui/x-data-grid";
+import { useAtom } from "jotai";
+import { useTranslation } from "react-i18next";
+import { errorAtom } from "../../atoms/error";
+
+/**
+ * Error dialog component.
+ */
+const ErrorDialog = () => {
+  const { t } = useTranslation();
+  const [error, setError] = useAtom(errorAtom);
+
+  const handleClose = () => setError(null);
+
+  return (
+    <Dialog fullWidth open={!!error} onClose={handleClose}>
```

Comment:
```suggestion
    <Dialog transitionDuration={0} fullWidth open={!!error} onClose={handleClose}>
```

Reply:
Adding transition duration of 0 will make quitting dialog better. Currently it disappears in 2 steps: 1. The error text disappears 2. the dialog disappears